x/slashing/signing_info.go

```diff
@@ -20,6 +20,21 @@ func (k Keeper) getValidatorSigningInfo(ctx sdk.Context, address sdk.ConsAddress
 	return
 }
 
+// Stored by *validator* address (not operator address)
+func (k Keeper) iterateValidatorSigningInfos(ctx sdk.Context, handler func(address sdk.ConsAddress, info ValidatorSigningInfo) (stop bool)) {
+	store := ctx.KVStore(k.storeKey)
+	iter := sdk.KVStorePrefixIterator(store, ValidatorSigningInfoKey)
+	defer iter.Close()
```

Comment:
I think for these iteration methods, we should close the iterator at the end.

Reply:
Hmm OK I don't think the overhead from using `defer` is substantial compared to the other things we do (iterating over Merklized stores...) but I can change it.